tests/common.go

```diff
@@ -2111,19 +2114,43 @@ func AfterEachTest(contexts []*scheduler.Context, ids ...int) {
 
 // SetClusterContext sets context to clusterConfigPath
 func SetClusterContext(clusterConfigPath string) error {
+	// an empty string indicates the default kubeconfig.
+	// This variable is used to clearly indicate that in logs
+	var clusterConfigPathOrDefault = clusterConfigPath
+	if clusterConfigPathOrDefault == "" {
+		clusterConfigPathOrDefault = "default"
+	}
+	log.InfoD("Switching context to [%s]", clusterConfigPathOrDefault)
+	if clusterConfigPath == CurrentClusterConfigPath {
+		log.InfoD("Switching context: The context is already [%s]", clusterConfigPathOrDefault)
+		return nil
+	}
```

Comment:
In this functions, I store the value of the current cluster config (kubeconfig which is active) in `CurrentClusterConfigPath`. So, the comparison of `clusterConfigPath` func arg happens with `CurrentClusterConfigPath`.
_But_, there's a case where if `clusterConfigPath` is `""` (empty string), then the entire function assumes that it is the default cluster (`KUBECONFIG` env var, afaik). The only **problem** is that if the string is empty, then it can confuse someone in logs when them see `"Switching context: The context is already []"`, which is why, I ensure the log message is complete, by making the message `"Switching context: The context is already [default]"` if the string is `""`.

Reply:
Instead of doing this 

```
var clusterConfigPathOrDefault = clusterConfigPath
	if clusterConfigPathOrDefault == "" {
		clusterConfigPathOrDefault = "default"
	}
```

You can simply do this

```
if clusterConfigPath == CurrentClusterConfigPath {
                 if CurrentClusterConfigPath == "" {
		// construct log message
	}
		log.InfoD("Switching context: The context is already [%s]", clusterConfigPathOrDefault)
		return nil
	}
```

Reply:
I fixed it to make it slightly more readable. I still had to keep it mostly the way it was, as another logging statement at the end of the function uses `clusterConfigPathForLog` (previously `clusterConfigPathOrDefault`), so in my code we only assign it once, which is better (than comparing twice).